### PR TITLE
7572_handle_event_genres

### PR DESCRIPTION
**Pull request by vietdung1996**
## [Pull request title]
7572_handle_event_genres
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7572

### CODE REVIEW

**Comment by vietdung1996** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,6 +11,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".ui.playmusic.PlayMusicActivity"></activity>
```
Comment (NguyenMinhTuanB) [inline app/src/main/AndroidManifest.xml]: convention 3.1

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -20,4 +22,4 @@
         </activity>
     </application>
 
-</manifest>
+</manifest>
```
Comment (NguyenMinhTuanB) [inline app/src/main/AndroidManifest.xml]: need an endline

File: app/src/main/java/com/example/dung/music_59/utils/StringUtils.java [hunk 3]
```diff
@@ -24,7 +24,7 @@ public static String initGenreApi(String kind, String keyGenre, int limit, int o
     }
 
     public static String initStreamApi(long trackId) {
-        String BASE_URL_STREAM = StringUtils.append(Constants.BASE_URL, Constants.TRACKS, "/%s/stream?", Constants.CLIENT_ID);
+        String BASE_URL_STREAM = StringUtils.append(Constants.BASE_URL_STREAM, Constants.TRACKS, "/%s/stream?", Constants.CLIENT_ID);
         return String.format(BASE_URL_STREAM,
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/dung/music_59/utils/StringUtils.java:24]: Đừng enter tùy tiện.

File: app/src/main/res/layout/activity_play_music.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.playmusic.PlayMusicActivity">
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_play_music.xml:8]: remove bl

File: app/src/main/res/layout/activity_play_music.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+</android.support.constraint.ConstraintLayout>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_play_music.xml]: tương tự

**Comment by vietdung1996** [comment]
ready

**Pull request merged**